modules/obtaining-ocp-packages-restricted-network-cluster.adoc

```diff
@@ -0,0 +1,182 @@
+----
+
+.. Attach a pool ID for a subscription that provides {product-title}:
++
+[source,terminal]
+----
+$ subscription-manager attach --pool=<pool_id>
+----
++
+[source,terminal]
+----
+$ subscription-manager repos --disable="*"
+----
+
+. Enable only the repositories required by {product-title} {product-version}.
++
+--
+** For cloud installations and on-premise installations on x86_64 servers, run the following command:
++
+[source,terminal]
+----
+# subscription-manager repos \
+    --enable="rhel-7-server-rpms" \
+    --enable="rhel-7-server-extras-rpms" \
+    --enable="rhel-7-server-ose-4.7-rpms" \
+    --enable="rhel-7-server-ansible-2.9-rpms"
+----
+--
+** For on-premise installations on IBM POWER8 servers, run the following command:
++
+[source,terminal]
+----
+# subscription-manager repos \
+    --enable="rhel-7-for-power-le-rpms" \
+    --enable="rhel-7-for-power-le-extras-rpms" \
+    --enable="rhel-7-for-power-le-optional-rpms" \
+    --enable="rhel-7-server-ansible-2.9-for-power-le-rpms" \
+    --enable="rhel-7-server-for-power-le-rhscl-rpms" \
+    --enable="rhel-7-for-power-le-ose-4.7-rpms"
+----
+** For on-premise installations on IBM POWER9 servers, run the following command:
++
+[source,terminal]
+----
+# subscription-manager repos \
+    --enable="rhel-7-for-power-9-rpms" \
+    --enable="rhel-7-for-power-9-extras-rpms" \
+    --enable="rhel-7-for-power-9-optional-rpms" \
+    --enable="rhel-7-server-ansible-2.9-for-power-9-rpms" \
+    --enable="rhel-7-server-for-power-9-rhscl-rpms" \
+    --enable="rhel-7-for-power-9-ose-4.7-rpms"
+----
+
+. Install required packages:
++
+[source,terminal]
+----
+$ sudo yum -y install yum-utils createrepo docker git
+----
++
+The `yum-utils` package provides the *reposync* utility. You can mirror yum repositories, and use the `createrepo` package to create a usable `yum` repository from a directory.
+
+. Make a directory to store the software in the server's storage or to a USB drive or other external device:
++
+[source,terminal]
+----
+$ mkdir -p </path/to/repos>
+----
++
+[IMPORTANT]
+====
+If you can re-connect this server to the disconnected LAN and use it as the repository server, store the files locally. If you cannot, use USB-connected storage so you can transport the software to a repository server in your disconnected LAN.
+====
+
+. Sync the packages and create the repository for each of them.
+** For on-premise installations on x86_64 servers, run the following command:
++
+[source,terminal]
+----
+$ for repo in \
+  rhel-7-server-rpms \
+  rhel-7-server-extras-rpms \
+  rhel-7-server-ansible-2.9-rpms \
+  rhel-7-server-ose-4.7-rpms
+do
+  reposync --gpgcheck -lm --repoid=${repo} --download_path=</path/to/repos> <1>
+  createrepo -v </path/to/repos/>${repo} -o </path/to/repos/>${repo} <1>
+done
+----
+<1> Provide the path to the directory you created.
+** For on-premise installations on IBM POWER8 servers, run the following command:
```

Comment:
In https://docs.openshift.com/container-platform/4.6/machine_management/adding-rhel-compute.html#rhel-preparing-playbook-machine_adding-rhel-compute, we did not mentioned non-x86 arch servers, so I think we should remove this part to keep consistent with install section